modules/ROOT/pages/bare-metal.adoc

```diff
@@ -177,28 +177,60 @@ Alternatively, instead of generating per-machine ISOs, you can have a `--pre-ins
 === Installing on iSCSI
 
 To install CoreOS on an iSCSI boot device, follow the same
-steps as described above to get the live environnement, simply adding the correct kernel arguments.
+steps as described above to get the live environnement, and add add the iSCSI related kernel arguments.
 
 
-- Mount the iscsi target from the live environnement :
+- Mount the iSCSI target from the live environment:
 [source, bash]
 ----
 sudo iscsiadm -m discovery -t st -p 127.0.0.1
 sudo iscsiadm -m node -T iqn.2023-10.coreos.target.vm:coreos -l
 ----
 
-- run `coreos-installer`:
+- Append the necessary kargs when running `coreos-installer`:
+
+.Installing to an iSCSI target with iBFT
 [source, bash]
-----
+---
+sudo coreos-installer install \
+    /dev/disk/by-path/ip-127.0.0.1\:3260-iscsi-iqn.2023-10.coreos.target.vm\:coreos-lun-0 \
+    --append-karg rd.iscsi.firmware=1 --append-karg ip=ibft \
+    --console ttyS0 \
+    --ignition-url https://example.com/example.ign
+---
+
+.Installing to an iSCSI target with manual configuration
+[source, bash]
+---
 sudo coreos-installer install \
     /dev/disk/by-path/ip-127.0.0.1\:3260-iscsi-iqn.2023-10.coreos.target.vm\:coreos-lun-0 \
     --append-karg rd.iscsi.initiator=iqn.2024-02.com.yourorg.name:lun0 \
     --append-karg netroot=iscsi:iqn.2023-10.coreos.target.vm:coreos` \
     --console ttyS0 \
     --ignition-url https://example.com/example.ign
+---
+
+
+NOTE: All this can also be  set using `coreos-installer iso customize` or `coreos-installer pxe customize`. (See "Customizing installation" section above).
+
+For example using iBFT:
+
+[source,bash,subs="attributes"]
+----
+# Create customized.iso which:
+# - Automatically mounts iscsi target using mount-iscsi.sh
+# - Provisions the installed system with config.ign
+# - Configures the installed GRUB and kernel to use iBFT
```

Comment:
Technically these don't configure GRUB, only the kernel. :) Maybe just

```suggestion
# - Configures the installed system to use iBFT
```

?

But on that topic, I guess it's implied that the user will need to have e.g. an iPXE script for actually booting up. I wonder if we should at least have a `NOTE` about it if not also a sample iPXE script?

Reply:
My thinking was " if you are doing iBFT you know what you are doing already " but we could add that, it's not like it's a big addition

Reply:
I added an iPXE boot script example.